en/deploy/README.md

```diff
@@ -24,7 +24,7 @@ Git tracks changes to a particular set of files in what's called a code reposito
 
 {% filename %}command-line{% endfilename %}
 ```
-$ git init
+$ git init -b main
```

Comment:
The `-b` option was introduced to `git init` with Git 2.28 in 2020-07-27. I guess we can't yet assume that everyone has that or a newer version of Git.

Reply:
@das-g  I agree.

Let's say I am a new user and I am using Ubuntu focal (20.04LTS). So I have git version 2.25.1. Now I executed following commands as in tutorial:

```
$ git init
$ git config --global user.name "Your Name"
$ git config --global user.email you@example.com
$ git add .
$ git commit -m "My Django Girls app, first commit"
$ git remote add origin https://github.com/<your-github-username>/my-first-blog.git
$ git push -u origin master
```

So now I assume there is a new branch called `master` in Github along with `main` branch.

I think the problem comes in Pythonanywhere.com:

(PythonAnywhere command-line)
```
$ pip3.6 install --user pythonanywhere
$ pa_autoconfigure_django.py --python=3.6 https://github.com/<your-github-username>/my-first-blog.git
```

Now the pythonanywhere script pulls the main branch from Github because this is the default branch set in Github. So, the command will propably result in "Could not find your settings.py".

**NOTE:** I have not tested this theory. But what if this happens? Correct me if I went wrong somewhere.

Reply:
I've tried that with a new GitHub account: https://github.com/das-g2/my-first-blog

That series of actions seems to set `master` as the new repo's default branch. (I didn't change that setting manually in the repo settings page.) So at least for now, that doesn't seem to be a problem.

Reply:
I think this might be the temporary fallback solution. This is provided to all user when a new repo is made on GIthub now.

```
echo "# my-first-blog" >> README.md
git init
git add README.md
git commit -m "first commit"
git branch -M main
git remote add origin git@github.com:<username>/my-first-blog.git
git push -u origin main
```

@das-g try check this

Reply:
> I've tried that with a new GitHub account: https://github.com/das-g2/my-first-blog
> 
> That series of actions seems to set `master` as the new repo's default branch. (I didn't change that setting manually in the repo settings page.) So at least for now, that doesn't seem to be a problem.

I just checked too. So nothing to worry. @das-g Should I close this pull request?

Reply:
I'm not sure. The more comprehensive `.gitignore` and dropping `--all` for `git add .` might still be worthwhile changes already now.

Reply:
@das-g check PR #1682